src/test/java/io/swagger/swaggerhub/plugin/utils/SwaggerHubUploadTestConstants.java

```diff
@@ -26,6 +26,8 @@
     public static final String SCM_ACCOUNT= "scm-account";
     public static final String SCM_PERSONAL_ACCESS_TOKEN = "scm-personal-access-token";
     public static final String SCM_PROJECT = "scm-project";
+    public static final String SCM_URL = "http://localhost:1234";
+    public static final String SCM_PROJECT_COLLECTION = "DefaultCollection";
```

Comment:
@michaelmelodysmartbear : Remove references to TFS in the constant property names. We may have other SCM integrations in future which use those same properties

Done